### ISSUE TITLE
Redis Crashing When Data is Added to Memory

### ISSUE THREAD

**Issue by remriel** [Significant-Gravitas/Auto-GPT#2870]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Windows

### Which version of Auto-GPT are you using?

Master (branch)

### GPT-3 or GPT-4?

GPT-3.5

### Steps to reproduce 🕹

Set up Redis using command in documentation
No password
Default port
Started docker container
Started auto-gpt

### Your prompt 📝

```yaml
ai_goals:
- Develop an intuitive web interface to explore the data and analytical insights
- Retrieve the latest economic data from an economic calendar for the last 6 months
- Assess the sentiment related to announcements from the Federal Reserve and leading
  figures in the finance and banking sectors from the last 6 months
- Analyze the collected data to generate economic forecasts for 3, 6, 12, and 24 months
  ahead
ai_name: Economy-GPT
ai_role: 'an AI for analyzing and predicting economic trends and creating web apps
  with Python and Flask '
```

### Your Logs 📒

```log
redis-stack-server
Continuous Mode:  ENABLED
WARNING:  Continuous mode is not recommended. It is potentially dangerous and may cause your AI to run forever or carry out actions you would not usually authorise. Use at your own risk.
GPT3.5 Only Mode:  ENABLED
NEWS:  Welcome to Auto-GPT! We'll keep you informed of the latest news and features by printing messages here. If you don't wish to see this message, you can run Auto-GPT with the --skip-news flag
Welcome back!  Would you like me to return to being Economy-GPT?
Continue with the last settings?
Name:  Economy-GPT
Role:  an AI for analyzing and predicting economic trends and creating web apps with Python and Flask
Goals: ['Develop an intuitive web interface to explore the data and analytical insights', 'Retrieve the most recent economic data from an economic calendar', 'Assess the sentiment related to announcements from the Federal Reserve and leading figures in the finance and banking sectors', 'Analyze the collected data to generate economic forecasts for 3, 6, 12, and 24 months ahead']
Continue (y/n):
Using memory of type:  RedisMemory
Using Browser:  chrome
 THOUGHTS:  I think the first step should be to retrieve the most recent economic data from an economic calendar. This will give us a good starting point for analyzing and predicting economic trends. We can then use this data to assess the sentiment related to announcements from the Federal Reserve and leading figures in the finance and banking sectors. Once we have a good understanding of the current economic landscape, we can begin analyzing the data to generate economic forecasts for 3, 6, 12, and 24 months ahead.
REASONING:  Retrieving the most recent economic data will give us a good starting point for analyzing and predicting economic trends. We can then use this data to assess the sentiment related to announcements from the Federal Reserve and leading figures in the finance and banking sectors. Once we have a good understanding of the current economic landscape, we can begin analyzing the data to generate economic forecasts for 3, 6, 12, and 24 months ahead.
PLAN:
-  Retrieve the most recent economic data from an economic calendar
-  Assess the sentiment related to announcements from the Federal Reserve and leading figures in the finance and banking sectors
-  Analyze the data to generate economic forecasts for 3, 6, 12, and 24 months ahead
CRITICISM:  I need to ensure that I am using reliable sources for the economic data and sentiment analysis. Additionally, I need to make sure that my analysis is thorough and takes into account all relevant factors.
NEXT ACTION:  COMMAND = browse_website ARGUMENTS = {'url': 'https://www.forexfactory.com/calendar.php', 'question': 'What is the most recent economic data?'}

DevTools listening on ws://127.0.0.1:53045/devtools/browser/2fe92543-4a9d-4a8c-a46c-ceec192dc9c4
[0421/192652.132:INFO:CONSOLE(98)] "Unrecognized feature: 'attribution-reporting'.", source: https://www.googletagservices.com/dcm/impl_v95.js (98)
[0421/192652.144:INFO:CONSOLE(98)] "Unrecognized feature: 'attribution-reporting'.", source: https://www.googletagservices.com/dcm/impl_v95.js (98)
Text length: 11436 characters
Adding chunk 1 / 1 to memory
Summarizing chunk 1 / 1 of length 11440 characters, or 4048 tokens
Added chunk 1 summary to memory, of length 159 characters
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\GPT\Auto-GPT-0.2.2\autogpt\__main__.py", line 5, in <module>
    autogpt.cli.main()
  File "C:\Python311\Lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\click\core.py", line 1635, in invoke
    rv = super().invoke(ctx)
         ^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\click\decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\GPT\Auto-GPT-0.2.2\autogpt\cli.py", line 177, in main
    agent.start_interaction_loop()
  File "D:\GPT\Auto-GPT-0.2.2\autogpt\agent\agent.py", line 213, in start_interaction_loop
    self.memory.add(memory_to_add)
  File "D:\GPT\Auto-GPT-0.2.2\autogpt\memory\redismem.py", line 101, in add
    pipe.execute()
  File "C:\Python311\Lib\site-packages\redis\client.py", line 2109, in execute
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\retry.py", line 49, in call_with_retry
    fail(error)
  File "C:\Python311\Lib\site-packages\redis\client.py", line 2111, in <lambda>
    lambda error: self._disconnect_raise_reset(conn, error),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "C:\Python311\Lib\site-packages\redis\client.py", line 2110, in <lambda>
    lambda: execute(conn, stack, raise_on_error),
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\client.py", line 1964, in _execute_transaction
    self.parse_response(connection, "_")
  File "C:\Python311\Lib\site-packages\redis\client.py", line 2049, in parse_response
    result = Redis.parse_response(self, connection, command_name, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\client.py", line 1275, in parse_response
    response = connection.read_response()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\connection.py", line 865, in read_response
    response = self._parser.read_response(disable_decoding=disable_decoding)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\connection.py", line 346, in read_response
    result = self._read_response(disable_decoding=disable_decoding)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\connection.py", line 356, in _read_response
    raw = self._buffer.readline()
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\redis\connection.py", line 259, in readline
    self._read_from_socket()
  File "C:\Python311\Lib\site-packages\redis\connection.py", line 212, in _read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
```

**Comment by enixlabs** [comment]
Are you using Docker Desktop or direct on WSL2 (Linux Variant)?

If using 'Docker Desktop', open it up and click on the redis-stack, it will open the logs and have information in them.

Post that info from the logs

**Comment by remriel** [comment]
2023-04-21 19:21:52 1:C 22 Apr 2023 02:21:52.316 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
2023-04-21 19:21:52 1:C 22 Apr 2023 02:21:52.316 # Redis version=6.2.10, bits=64, commit=00000000, modified=0, pid=1, just started
2023-04-21 19:21:52 1:C 22 Apr 2023 02:21:52.316 # Configuration loaded
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.317 * monotonic clock: POSIX clock_gettime
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.318 * Running mode=standalone, port=6379.
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.318 # Server initialized
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.318 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.328 * <search> Redis version found by RedisSearch : 6.2.10 - oss
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.328 * <search> RediSearch version 2.6.7 (Git=HEAD-a3647123)
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.328 * <search> Low level api version 1 initialized successfully
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.328 * <search> concurrent writes: OFF, gc: ON, prefix min length: 2, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  1000000, search pool size: 20, index pool size: 8, 
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.329 * <search> Initialized thread pool!
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.329 * <search> Enabled diskless replication
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.329 * <search> Enabled role change notification
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.329 * Module 'search' loaded from /usr/lib/redis/modules/redisearch.so
2023-04-21 19:21:52 1:M 22 Apr 2023 02:21:52.329 * Ready to accept connections
2023-04-21 19:21:55 1:M 22 Apr 2023 02:21:55.489 # <search> creating vector index. Server memory limit: 13392916480B, required memory: 6470160B, available memory: 13391974624B
2023-04-21 19:22:43 
2023-04-21 19:22:43 
2023-04-21 19:22:43 === REDIS BUG REPORT START: Cut & paste starting from here ===
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.456 # Redis 6.2.10 crashed by signal: 4, si_code: 2
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.456 # Crashed running the instruction at: 0x7fde4c03e789
2023-04-21 19:22:43 
2023-04-21 19:22:43 ------ STACK TRACE ------
2023-04-21 19:22:43 EIP:
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(_Z35FP32_InnerProductSIMD16Ext_SSE_implPKvS0_m+0x9)[0x7fde4c03e789]
2023-04-21 19:22:43 
2023-04-21 19:22:43 Backtrace:
2023-04-21 19:22:43 /lib/x86_64-linux-gnu/libpthread.so.0(+0x13140)[0x7fde4cdc6140]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(invalidFunctionWasCalled+0x0)[0x55e0c8b10290]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(_Z30FP32_InnerProductSIMD16Ext_SSEPKvS0_m+0x9)[0x7fde4c03e809]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(_ZNK9HNSWIndexIffE11searchLayerEjPKvmm+0x123)[0x7fde4c02bba3]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(_ZN9HNSWIndexIffE12appendVectorEPKvm+0x301)[0x7fde4c032061]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(VecSimIndex_AddVector+0x70)[0x7fde4c009970]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(IndexerBulkAdd+0x137)[0x7fde4bf78d37]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(+0xdabd2)[0x7fde4bf8fbd2]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(+0xdb1b0)[0x7fde4bf901b0]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(Indexer_Add+0x55)[0x7fde4bf90b05]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(Document_AddToIndexes+0xf1)[0x7fde4bf79061]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(IndexSpec_UpdateDoc+0x140)[0x7fde4bfbf6b0]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(Indexes_UpdateMatchingWithSchemaRules+0xd6)[0x7fde4bfbfd96]
2023-04-21 19:22:43 /usr/lib/redis/modules/redisearch.so(HashNotificationCallback+0x138)[0x7fde4bf9c2c8]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(moduleNotifyKeyspaceEvent+0xce)[0x55e0c8b4cd9e]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(notifyKeyspaceEvent+0x21)[0x55e0c8b39671]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(hsetCommand+0xf9)[0x55e0c8b039c9]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(call+0xdb)[0x55e0c8abf29b]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(execCommand+0x263)[0x55e0c8b0fed3]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(call+0xdb)[0x55e0c8abf29b]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(processCommand+0x5a3)[0x55e0c8ac0eb3]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(processInputBuffer+0xf8)[0x55e0c8ad46c8]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(+0xfc668)[0x55e0c8b6c668]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(aeProcessEvents+0x292)[0x55e0c8ab7e62]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(aeMain+0x1d)[0x55e0c8ab80cd]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(main+0x316)[0x55e0c8ab42e6]
2023-04-21 19:22:43 /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xea)[0x7fde4cc01d0a]
2023-04-21 19:22:43 redis-server 0.0.0.0:6379(_start+0x2a)[0x55e0c8ab47ba]
2023-04-21 19:22:43 
2023-04-21 19:22:43 ------ REGISTERS ------
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # 
2023-04-21 19:22:43 RAX:00007ffd4d80af60 RBX:00007fde4c75a388
2023-04-21 19:22:43 RCX:000055e0ca71e758 RDX:0000000000000600
2023-04-21 19:22:43 RDI:00007ffd4d809760 RSI:00007fde43580554
2023-04-21 19:22:43 RBP:00007ffd4d8096d8 RSP:00007ffd4d809638
2023-04-21 19:22:43 R8 :0000000000000000 R9 :00000000000000c8
2023-04-21 19:22:43 R10:0000000000000001 R11:00007fde43581de8
2023-04-21 19:22:43 R12:00007ffd4d80b0f0 R13:0000000000000000
2023-04-21 19:22:43 R14:00007ffd4d80b108 R15:0000000000000001
2023-04-21 19:22:43 RIP:00007fde4c03e789 EFL:0000000000010287
2023-04-21 19:22:43 CSGSFS:002b000000000033
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809647) -> 00007fde43da02d0
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809646) -> 0000000000000018
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809645) -> 0000000000000010
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809644) -> 00007fde4c1fc6b8
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809643) -> 0000000000000020
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809642) -> 00007ffd4d8096c0
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809641) -> 00007ffd4d8096b0
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809640) -> 0000000000000000
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963f) -> 00007ffd4d809760
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963e) -> 00000000000000c8
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963d) -> 0000000000000000
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963c) -> 0000000000000000
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963b) -> 0000000000000000
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d80963a) -> 00007fde4c02bba3
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809639) -> 00007ffd4d8096d8
2023-04-21 19:22:43 1:M 22 Apr 2023 02:22:43.458 # (00007ffd4d809638) -> 00007fde4c03e809
2023-04-21 19:22:43 
2023-04-21 19:22:43 ------ INFO OUTPUT ------
2023-04-21 19:22:43 # Server
2023-04-21 19:22:43 redis_version:6.2.10
2023-04-21 19:22:43 redis_git_sha1:00000000
2023-04-21 19:22:43 redis_git_dirty:0
2023-04-21 19:22:43 redis_build_id:6d9bedf3bdf8dfa0
2023-04-21 19:22:43 redis_mode:standalone
2023-04-21 19:22:43 os:Linux 5.10.16.3-microsoft-standard-WSL2 x86_64
2023-04-21 19:22:43 arch_bits:64
2023-04-21 19:22:43 monotonic_clock:POSIX clock_gettime
2023-04-21 19:22:43 multiplexing_api:epoll
2023-04-21 19:22:43 atomicvar_api:c11-builtin
2023-04-21 19:22:43 gcc_version:10.2.1
2023-04-21 19:22:43 process_id:1
2023-04-21 19:22:43 process_supervised:no
2023-04-21 19:22:43 run_id:4b877f2c6a8d9539d3b4377175b7f41d786e40ff
2023-04-21 19:22:43 tcp_port:6379
2023-04-21 19:22:43 server_time_usec:1682130163455946
2023-04-21 19:22:43 uptime_in_seconds:51
2023-04-21 19:22:43 uptime_in_days:0
2023-04-21 19:22:43 hz:10
2023-04-21 19:22:43 configured_hz:10
2023-04-21 19:22:43 lru_clock:4408563
2023-04-21 19:22:43 executable:/data/redis-server
2023-04-21 19:22:43 config_file:/usr/local/etc/redis/redis.conf
2023-04-21 19:22:43 io_threads_active:0
2023-04-21 19:22:43 
2023-04-21 19:22:43 # Clients
2023-04-21 19:22:43 connected_clients:1
2023-04-21 19:22:43 cluster_connections:0
2023-04-21 19:22:43 maxclients:10000
2023-04-21 19:22:43 client_recent_max_input_buffer:24
2023-04-21 19:22:43 client_recent_max_output_buffer:0
2023-04-21 19:22:43 blocked_clients:0
2023-04-21 19:22:43 tracking_clients:0
2023-04-21 19:22:43 clients_in_timeout_table:0
2023-04-21 19:22:43 
2023-04-21 19:22:43 # Memory
2023-04-21 19:22:43 used_memory:8899776
2023-04-21 19:22:43 used_memory_human:8.49M
2023-04-21 19:22:43 used_memory_rss:17494016
2023-04-21 19:22:43 used_memory_rss_human:16.68M
2023-04-21 19:22:43 used_memory_peak:8899776
2023-04-21 19:22:43 used_memory_peak_human:8.49M
2023-04-21 19:22:43 used_memory_peak_perc:100.12%
2023-04-21 19:22:43 used_memory_overhead:899680
2023-04-21 19:22:43 used_memory_startup:879032
2023-04-21 19:22:43 used_memory_dataset:8000096
2023-04-21 19:22:43 used_memory_dataset_perc:99.74%
2023-04-21 19:22:43 allocator_allocated:9524896
2023-04-21 19:22:43 allocator_active:9842688
2023-04-21 19:22:43 allocator_resident:12742656
2023-04-21 19:22:43 total_system_memory:13392916480
2023-04-21 19:22:43 total_system_memory_human:12.47G
2023-04-21 19:22:43 used_memory_lua:30720
2023-04-21 19:22:43 used_memory_lua_human:30.00K
2023-04-21 19:22:43 used_memory_scripts:0
2023-04-21 19:22:43 used_memory_scripts_human:0B
2023-04-21 19:22:43 number_of_cached_scripts:0
2023-04-21 19:22:43 maxmemory:0
2023-04-21 19:22:43 maxmemory_human:0B
2023-04-21 19:22:43 maxmemory_policy:noeviction
2023-04-21 19:22:43 allocator_frag_ratio:1.03
2023-04-21 19:22:43 allocator_frag_bytes:317792
2023-04-21 19:22:43 allocator_rss_ratio:1.29
2023-04-21 19:22:43 allocator_rss_bytes:2899968
2023-04-21 19:22:43 rss_overhead_ratio:1.37
2023-04-21 19:22:43 rss_overhead_bytes:4751360
2023-04-21 19:22:43 mem_fragmentation_ratio:1.98
2023-04-21 19:22:43 mem_fragmentation_bytes:8665520
2023-04-21 19:22:43 mem_not_counted_for_evict:0
2023-04-21 19:22:43 mem_replication_backlog:0
2023-04-21 19:22:43 mem_clients_slaves:0
2023-04-21 19:22:43 mem_clients_normal:20496
2023-04-21 19:22:43 mem_aof_buffer:0
2023-04-21 19:22:43 mem_allocator:jemalloc-5.1.0
2023-04-21 19:22:43 active_defrag_running:0
2023-04-21 19:22:43 lazyfree_pending_objects:0
2023-04-21 19:22:43 lazyfreed_objects:0
2023-04-21 19:22:43 
2023-04-21 19:22:43 # Persistence
2023-04-21 19:22:43 loading:0
2023-04-21 19:22:43 current_cow_size:0
2023-04-21 19:22:43 current_cow_size_age:0
2023-04-21 19:22:43 current_fork_perc:0.00
2023-04-21 19:22:43 current_save_keys_processed:0
2023-04-21 19:22:43 current_save_keys_total:0
2023-04-21 19:22:43 rdb_changes_since_last_save:5
2023-04-21 19:22:43 rdb_bgsave_in_progress:0
2023-04-21 19:22:43 rdb_last_save_time:1682130112
2023-04-21 19:22:43 rdb_last_bgsave_status:ok
2023-04-21 19:22:43 rdb_last_bgsave_time_sec:-1
2023-04-21 19:22:43 rdb_current_bgsave_time_sec:-1
2023-04-21 19:22:43 rdb_last_cow_size:0
2023-04-21 19:22:43 aof_enabled:0
2023-04-21 19:22:43 aof_rewrite_in_progress:0
2023-04-21 19:22:43 aof_rewrite_scheduled:0
2023-04-21 19:22:43 aof_last_rewrite_time_sec:-1
2023-04-21 19:22:43 aof_current_rewrite_time_sec:-1
2023-04-21 19:22:43 aof_last_bgrewrite_status:ok
2023-04-21 19:22:43 aof_last_write_status:ok
2023-04-21 19:22:43 aof_last_cow_size:0
2023-04-21 19:22:43 module_fork_in_progress:0
2023-04-21 19:22:43 module_fork_last_cow_size:0
2023-04-21 19:22:43 
2023-04-21 19:22:43 # Stats
2023-04-21 19:22:43 total_connections_received:1
2023-04-21 19:22:43 total_commands_processed:11
2023-04-21 19:22:43 instantaneous_ops_per_sec:0
2023-04-21 19:22:43 total_net_input_bytes:24763
2023-04-21 19:22:43 total_net_output_bytes:1284
2023-04-21 19:22:43 instantaneous_input_kbps:0.00
2023-04-21 19:22:43 instantaneous_output_kbps:0.00
2023-04-21 19:22:43 rejected_connections:0
2023-04-21 19:22:43 sync_full:0
2023-04-21 19:22:43 sync_partial_ok:0
2023-04-21 19:22:43 sync_partial_err:0
2023-04-21 19:22:43 expired_keys:0
2023-04-21 19:22:43 expired_stale_perc:0.00
2023-04-21 19:22:43 expired_time_cap_reached_count:0
2023-04-21 19:22:43 expire_cycle_cpu_milliseconds:1
2023-04-21 19:22:43 evicted_keys:0
2023-04-21 19:22:43 keyspace_hits:2
2023-04-21 19:22:43 keyspace_misses:1

**Comment by ntindle** [comment]
Seems to be a setup issue, please use the #tech-support channel in discord
